exporters/etw/include/opentelemetry/exporters/etw/etw_tracer.h

```diff
@@ -250,8 +250,41 @@ class Tracer : public opentelemetry::trace::Tracer,
     auto spanContext = spanBase.GetContext();
 
     // Populate Span with presaved attributes
-    Span &currentSpan   = const_cast<Span &>(span);
-    Properties evt      = GetSpanAttributes(currentSpan);
+    Span &currentSpan = const_cast<Span &>(span);
+    Properties evt    = GetSpanAttributes(currentSpan);
+
+#if defined(ENABLE_ENV_PROPERTIES)
+
+    Properties env_properties_env = {};
+    if (evt.size() > 0)
+    {
+      bool has_customer_attribute = false;
+      nlohmann::json env_properties_json = nlohmann::json::object();
+      for (auto &kv : evt)
+      {
+        nostd::string_view key = kv.first.data();
+
+        // don't serialize fields propagated from span data.
+        if (key == ETW_FIELD_NAME || key == ETW_FIELD_SPAN_ID || key == ETW_FIELD_TRACE_ID ||
+            key == ETW_FIELD_SPAN_PARENTID)
```

Comment:
what should be the behavior if key is `env_properties` - extract the value from it, and append it to existing `env_properties` ? Current behavior looks good to if we document that.

Reply:
Yes, `env_properties` set by user will be nested. I think this is expected and has parity with Linux, because `Span::SetAttribute` on Linux doesn't parse the attribute name with special handling.

Are you aware of any appropriate document (link) for this?

Reply:
No, I think it should be fine to be consistent with fluentd.